runners/cs_runner/config.R

```diff
@@ -0,0 +1,90 @@
+# load required packages
+pacman::p_load(
```

Comment:
please attach `hydrofabric` and remove aws.s3, sf, terra, glue here. Can we also get by without logger? Just one less package to manage? Up to you

Reply:
Happy to drop `logger` and will drop those other packages too